doc/style.css

```diff
@@ -1,267 +1,267 @@
 html {
-    line-height: 1.15;
-    -webkit-text-size-adjust: 100%;
+	line-height: 1.15;
+	-webkit-text-size-adjust: 100%;
```

Comment:
spaces -> tabs could be controversial and seems an unnecessary diff (IMO).

Could we configure biome to either ignore css for now, or use spaces?

What (if anything) do we have configured in our editorconfig for css indentation?